src/components/Borrow/SwitchBorrow/ActionBorrow.tsx

```diff
@@ -0,0 +1,420 @@
+        <Divider className={classes.divider} />
+        <Grid className={classes.bottomGrid}>
+          <Grid className={classes.bottomInfo}>
+            <Grid>
+              <Typography className={classes.infoTitle}>Interest rate:</Typography>
+              <Typography className={classes.infoValueFrom}>
+                <AnimatedNumber
+                  value={interestRate}
+                  formatValue={(value: number) => value.toFixed(3)}
+                />
+                %
+              </Typography>
+            </Grid>
+            <Grid>
+              <Typography className={classes.infoTitle}>Liquidation price:</Typography>
+
+              <Grid container alignItems='center'>
+                <Typography className={classes.infoValueTo}>{liquidationPriceTo}$</Typography>
+                <FlatIcon
+                  className={classes.flatIcon}
+                  style={{
+                    color:
+                      liquidationPriceTo >= liquidationPriceFrom
+                        ? colors.green.button
+                        : colors.red.error
+                  }}
+                />
+                <Typography className={classes.infoValueFrom}>
+                  <AnimatedNumber
+                    value={liquidationPriceFrom}
+                    formatValue={(value: number) => value.toFixed(3)}
+                  />{' '}
+                  $
+                </Typography>
+              </Grid>
+            </Grid>
+            <Grid>
+              <Typography className={classes.infoTitle}>Collateral ratio:</Typography>
+              <Grid container>
+                <Typography className={classes.infoValueTo}>
+                  {collateralRatioTo.toFixed(2)}%
+                </Typography>
+                <FlatIcon
+                  className={classes.flatIcon}
+                  style={{
+                    color: cRatio <= 100 ? colors.green.button : colors.red.error
+                  }}
+                />
+                <Typography className={classes.infoValueFrom}>
+                  {' '}
+                  <AnimatedNumber
+                    value={collateralRatioFrom}
+                    formatValue={(value: number) => value.toFixed(2)}
+                  />
+                  %
+                </Typography>
+              </Grid>
+            </Grid>
+          </Grid>
+          <Grid className={classes.buttonAction} container>
+            <Progress state={getProgressState()} message={getProgressMessage()} />
+            <OutlinedButton
+              name={nameButton}
+              color='secondary'
+              className={classes.actionButton}
+              onClick={onClickSubmitButton}
+            />
+          </Grid>
+        </Grid>
+      </Grid>
+      <BorrowTable
+        collateral={pairIndex !== null ? pairs[pairIndex].collateralData.symbol : 'WSOL'}
+        borrowed={pairIndex !== null ? pairs[pairIndex].syntheticData.symbol : 'WSOL'}
+        currentDebt={10}
```

Comment:
there is still hardcoded data in this component

Reply:
ok i corrected it.